Linked_list/Linked_List.c

```diff
@@ -1,122 +1,138 @@
+// Forward Linked List
 #include<stdio.h>
 #include<stdlib.h>
-// Forward Linked List
-struct List{
+
+struct List
+{
 	int value ;  
 	struct List *node; 
```

Comment:
Woud not be beter if you rename *node to next_node or something similar? It's just a matter of a better reading an understanding of the code.